src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonConverterAttribute.cs

```diff
@@ -5,16 +5,16 @@
 namespace System.Text.Json.Serialization
 {
     /// <summary>
-    /// When placed on a property or type, specifies the converter type to use.
+    /// When placed on a property, field or type, specifies the converter type to use.
     /// </summary>
     /// <remarks>
     /// The specified converter type must derive from <see cref="JsonConverter"/>.
-    /// When placed on a property, the specified converter will always be used.
+    /// When placed on a member, the specified converter will always be used.
```

Comment:
```suggestion
    /// When placed on a property or field, the specified converter will always be used.
```

Let's use "property or field" rather than "member" here and for similar occurrences, since this is not applicable to other members like methods and indexers.